tsl/src/continuous_aggs/refresh.c

```diff
@@ -654,6 +657,22 @@ continuous_agg_refresh(PG_FUNCTION_ARGS)
 	cagg = cagg_get_by_relid_or_fail(cagg_relid);
 	refresh_window.type = cagg->partition_type;
 
+	/*
+	 * If we are forcing the refresh range, then we also force the
+	 * move. Otherwise, we check if there is a policy defined that moves the
+	 * invalidations to the materialization invalidation log.
+	 *
+	 * Right now it is not possible to just force the hypertable invalidations
+	 * to be moved and do a normal refresh.
+	 */
+	if (!force)
+	{
+		List *jobs = ts_bgw_job_find_by_proc_and_hypertable_id(POLICY_PROCESS_HYPER_INVAL_PROC_NAME,
+															   FUNCTIONS_SCHEMA_NAME,
+															   cagg->data.raw_hypertable_id);
+		move_hypertable_invalidations = list_length(jobs) == 0;
+	}
```

Comment:
We might want to have a separate option so that you can force-refresh a range but not process hypertable invalidations.